libs/designsystem/src/lib/components/chart/chart.component.spec.ts

```diff
@@ -24,10 +25,6 @@ describe('ChartComponent', () => {
     fixture.detectChanges();
   });
 
-  it('should create', () => {
```

Comment:
Why delete?

Reply:
Useless test, is covered by every other case that the component can be created.